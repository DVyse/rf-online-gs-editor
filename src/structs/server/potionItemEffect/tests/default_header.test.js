import defaultHeader from '../default_header';

describe('Structs', () => {
  describe('Server', () => {
    describe('PotionItemEffect', () => {
      describe('DefaultHeader', () => {
        it('should exist', () => {
          expect(
            defaultHeader.map(fld => fld.toPlainObject()),
          ).toMatchSnapshot();
        });
      });
    });
  });
});
