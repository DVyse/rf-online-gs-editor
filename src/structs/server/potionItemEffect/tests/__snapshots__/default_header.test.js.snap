// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Structs Server PotionItemEffect DefaultHeader should exist 1`] = `
Array [
  Object {
    "as": undefined,
    "len": 32,
    "name": "nCount",
    "schemaProps": Object {},
    "type": [Function],
    "typeClassName": "Number",
    "weight": 4,
  },
  Object {
    "as": undefined,
    "len": 32,
    "name": "nCountColumns",
    "schemaProps": Object {},
    "type": [Function],
    "typeClassName": "Number",
    "weight": 4,
  },
  Object {
    "as": undefined,
    "len": 32,
    "name": "nBlockSize",
    "schemaProps": Object {},
    "type": [Function],
    "typeClassName": "Number",
    "weight": 4,
  },
]
`;
